sdk/keyvault/azure-security-keyvault-certificates/README.md

```diff
@@ -235,7 +233,7 @@ The following sections provide several code snippets covering some of the most c
 ### Create a Certificate Asynchronously
```

Comment:
[here](https://github.com/Azure/azure-sdk-for-java/pull/6061/files#diff-824e08966bb186e6f79351c1a042d490L191) should be `updateCertificateProperties`

Reply:
updated.